### DISCUSSION TITLE
Usage of Verovio in Wix

### DISCUSSION THREAD

**Discussion by GimmeSomeJazz** [rism-digital/verovio#3144] [Q&A] [answered]
Hi all!

I struggle to integrate Verovio in a Wix website and to use it, e.g.:

```
import verovio from 'public/verovio-toolkit-wasm.js';

$w.onReady(function () {
    var tk = new verovio.toolkit();
    ...
}
```

Any tips that would help me? Anyone did succeed to do such a thing?

Thanks you!

Maxime

**Comment by lpugin** [comment]
Maybe you can try using the VerovioApp? This is probably the simplest way to have basic rendering integrated in a web page.
https://www.verovio.org/app.html

**Reply by GimmeSomeJazz** [reply]
unfortunately the web app does not load in Wix :(

**Comment by ahankinson** [comment, marked as answer]
Could you post a link to your webpage where it is not working?

**Reply by GimmeSomeJazz** [reply]
don't know how to do that!

**Reply by ahankinson** [reply]
I'm sorry we can't help you more, but without seeing anything it's pretty hard to help fix it. Maybe try asking the Wix support people?
